packages/core/src/middleware/koa-quota-guard.ts

```diff
@@ -3,17 +3,25 @@ import type { MiddlewareType } from 'koa';
 import { type QuotaLibrary } from '#src/libraries/quota.js';
 import { type FeatureQuota } from '#src/utils/subscription/types.js';
 
+type Method = 'GET' | 'POST' | 'PUT' | 'DELETE';
```

Comment:
This should have a built in type in Node or Koa

Reply:
I'm afraid no type for method union from either Koa or Node. There is one from `got` though